packages/app/src/cli/services/context.ts

```diff
@@ -227,7 +227,7 @@ function includeConfigOnDeployPrompt(configPath: string): Promise<boolean> {
   return renderConfirmationPrompt({
     message: `Include \`${basename(
       configPath,
-    )}\` configuration on \`deploy\`? Soon, this will no longer be an option, and configuration will need to be included on deploy every time.`,
+    )}\` configuration on \`deploy\`? Soon, this will no longer be optional, and configuration will be included on every deploy.`,
```

Comment:
```suggestion
    )}\` configuration on \`deploy\`? Soon, this will no longer be optional and configuration will be included on every deploy.`,
```
Minor grammatical nitpick